omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -268,13 +268,15 @@ protected PageableResult doSearch(RequestContext context) throws ResponseExcepti
 			if (context.getType() != null) {
 				filterByType(orders, context.getType());
 			}
-			
+
+			// if the user indicated a specific sort parameter, apply sort here
 			if (StringUtils.isNotBlank(sortParam)) {
 				List<Order> sortedOrder = sortOrdersBasedOnDateActivatedOrDateStopped(orders, sortParam, status);
 				return new NeedsPaging<Order>(sortedOrder, context);
 			}
 			else {
-				return new NeedsPaging<Order>(orders, context);
+				List<Order> descSortedOrder = sortOrdersInDescendingOrderByDate(orders);
```

Comment:
Shouldn't this still call `sortOrdersBasedOnDateActivatedOrDateStopped()` albeit with `DESC` as the `sortParam`?